Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -126,12 +126,32 @@ public static int MaxMemoryCacheSize
         }
 
         /// <summary>
-        /// Load a specific image from the cache. If the image is not in the cache, ImageCache will try to download and store it.
+        /// Load a specific image from the cache. If the image is not in the cache, ImageCache will try to download and store it. If imagge cannot be loaded this method will return null.
         /// </summary>
         /// <param name="uri">Uri of the image.</param>
         /// <returns>a BitmapImage</returns>
         public static async Task<BitmapImage> GetFromCacheAsync(Uri uri)
         {
+            BitmapImage image = null;
+            try
+            {
+                image = await GetFromCacheOrDieAsync(uri);
+            }
+            catch
+            {
+                return null;
```

Comment:
This probably isn't needed. If you've handled a thrown exception, then image would still be null, which you are returning anyway.